code/modules/mob/mob.dm

```diff
@@ -1502,7 +1502,12 @@
 	else
 		living_flags |= QUEUE_NUTRITION_UPDATE
 
-///Apply a proper movespeed modifier based on items we have equipped
+/// Update equipment when something is equipped/dropped, to be overriden by children
```

Comment:
I will say `Update equipment when something happens` isn't very clear. 
Maybe something like `Update aspects of the mob based on what equipment we are wearing`